include/seqan/index/index_qgram.h

```diff
@@ -225,13 +225,13 @@ struct Fibre< Index<TText, IndexQGram<TShapeSpec, TSpec> >, FibreBucketMap>
     typedef Nothing Type;
 };
 
-#ifdef PLATFORM_WINDOWS_VS
+#ifdef STD_LIB_VS
 #pragma warning( push )
 // Disable warning C4521 locally (multiple copy constructors).
 #pragma warning( disable: 4521 )
 // Disable warning C4522 locally (multiple assignment operators).
 #pragma warning( disable: 4522 )
-#endif  // PLATFORM_WINDOWS_VS
+#endif  // STD_LIB_VS
```

Comment:
we have so many errors on windows, this change may not be noticeable.

Should I remove them?